ocaml/src/albamgr_protocol.ml

```diff
@@ -1017,12 +1040,12 @@ module Protocol = struct
            Nsm_host.to_buffer
            Llio.int64_to)
     | ListAvailableOsds ->
-      Llio.counted_list_to Osd.to_buffer
+      Llio.counted_list_to (OsdInfo.to_buffer ~version:2)
```

Comment:
Hmm, this is a bit unexpected. Due to the `~version:2` old clients will no longer be able to interpret the result of this call when talking to a new server.
Shouldn't that have shown up in the tests you've added?
The same remark applies to all places below where you added ~version:2 to the serialisation.
I expected ListAvailableOsds2, ListOsdsByOsdId2, ... operations to be added.